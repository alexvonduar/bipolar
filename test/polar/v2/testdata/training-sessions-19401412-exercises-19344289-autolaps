
A
'
	 Ó	 ÓfNF%0~?D-ŒCCD0
¢µJ
’≈∫@V^A
	 Ó	 Ó