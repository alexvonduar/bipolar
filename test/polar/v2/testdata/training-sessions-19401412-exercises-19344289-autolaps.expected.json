{
    "laps": [
        {
            "header": [
                {
                    "ascent": [
                        765.9716796875
                    ],
                    "descent": [
                        781.0594482421875
                    ],
                    "distance": [
                        8467.599609375
                    ],
                    "duration": [
                        {
                            "hours": [
                                1
                            ],
                            "milliseconds": [
                                750
                            ],
                            "minutes": [
                                27
                            ],
                            "seconds": [
                                2
                            ]
                        }
                    ],
                    "lap-type": [
                        3
                    ],
                    "split-time": [
                        {
                            "hours": [
                                1
                            ],
                            "milliseconds": [
                                750
                            ],
                            "minutes": [
                                27
                            ],
                            "seconds": [
                                2
                            ]
                        }
                    ]
                }
            ],
            "stats": [
                {
                    "heartrate": [
                        {
                            "average": [
                                162
                            ],
                            "maximum": [
                                181
                            ],
                            "minimum": [
                                74
                            ]
                        }
                    ],
                    "speed": [
                        {
                            "average": [
                                5.8366494178771973
                            ],
                            "maximum": [
                                13.895998954772949
                            ]
                        }
                    ]
                }
            ]
        }
    ],
    "summary": [
        {
            "average-duration": [
                {
                    "hours": [
                        1
                    ],
                    "milliseconds": [
                        750
                    ],
                    "minutes": [
                        27
                    ],
                    "seconds": [
                        2
                    ]
                }
            ],
            "best-duration": [
                {
                    "hours": [
                        1
                    ],
                    "milliseconds": [
                        750
                    ],
                    "minutes": [
                        27
                    ],
                    "seconds": [
                        2
                    ]
                }
            ]
        }
    ]
}
